synapse/tools/cryo/cat.py

```diff
@@ -47,7 +52,10 @@ def main(argv, outp=s_output.stdout):
         return 0
 
     for item in cryo.slice(path, opts.offset, opts.size, opts.timeout):
-        outp.printf(pprint.pformat(item))
+        if opts.jsonl:
+            outp.printf(json.dumps(item, sort_keys=True))
```

Comment:
The item from the ``sclie()`` api is an ``(offset, item)`` tuple. We probably want to unpack this to only write that inner item.

Reply:
ok